.travis/Dockerfile

```diff
@@ -61,33 +61,8 @@ RUN rosdep init \
 # clone source
 ENV ROS2_WS /root/ros2_ws
 RUN mkdir -p $ROS2_WS/src
-COPY ./workspace.repos $ROS2_WS
```

Comment:
I would still like to have the `.repos` file in it. Reasons for it are to resolve dependencies which are not yet resolvable via `rosdep`. One example for it is `ros_control` where travis would need to build `ros_control` as well as `ros_controllers` to build the latter correctly.

Reply:
I definitely see that this can be useful/necessary in some situations.

In this repo it is currently not necessary. If we keep a repos file, I think we will also move that logic out of the Dockerfile as we most likely want to use the repos file of the branch we test rather than the one from the  master branch.

While I would prefer not adding dead code to the CI scripts, I can add the logic in this PR and keep only an empty `workspace.repos` file for now if you prefer

Reply:
yeah, I like that idea. For context, I would like to have a dedicated repo on github for this travis integration, so that I "just" have to clean the logic into each repo and have a single point where we would have to maintain the travis logic, if this makes sense :)

Reply:
> I think we will also move that logic out of the Dockerfile as we most likely want to use the repos file of the branch we test rather than the one from the master branch

actually keeping the logic in the dockefile is totally fine :+1: I added back an empty workspace.repos file in https://github.com/Karsten1987/roscon2018/pull/13/commits/2e39c1096644f3851b3ce9e3517a8576da94957c

> For context, I would like to have a dedicated repo on github for this travis integration, so that I "just" have to clean the logic into each repo and have a single point where we would have to maintain the travis logic, if this makes sense :)

Yeah that's a great idea.
We could even go slightly further and have a set of files in that repo to have travis config for Linux and Mac as well as an appveyor file for windows and we could reuse that logic in any repo.
Regarding the docker image, maybe we could have a docker image with the "ros2 dev tools" already built and available on dockerhub. that way we could just pull it and use in all repos. That would save significant time as well